### PR TITLE
Add Classification task and model.

### PR DESCRIPTION
**Pull request by boru-roylu**
1. Add pos_weights in loss function to weight positive and negative examples for imbalanced ITM task.
2. Add Classification task (finetuning task).
3. Add mmt_classification_model.py.

### CODE REVIEW

File: src/modeling/models/mmt_classification_model.py [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+import tensorflow as tf
+from official.modeling import tf_utils
+from official.nlp.keras_nlp import layers
+
+
+class MmtClassificationModel(tf.keras.Model):
+  """Multimodal Transformer-based classification model.
+
+  Args:
```
Comment (santiaontanon-google) [inline src/modeling/models/mmt_classification_model.py]: nit: this docstring with the "Args", etc. belongs to the constructor, not to the class, right?

**Review by santiaontanon-google — APPROVED** [review]
Thanks Roy, LGTM except for a small nit about a docstring

File: src/modeling/models/mmt_classification_model.py (hunk 1 above)
Comment (boru-roylu) [reply src/modeling/models/mmt_classification_model.py]: Yeah! Edited! Thanks, Santi!

File: src/modeling/models/mmt_classification_model.py [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+        'encoder': encoder,
+        'classification_heads': classification_heads,
+        'name': name,
+    }
+    self.encoder = encoder
+    self.classification_heads = classification_heads or []
+    if len(set([cls.name for cls in self.classification_heads])) != len(
+        self.classification_heads):
+      raise ValueError('Classification heads should have unique names.')
+    assert len(self.classification_heads) > 0, 'At least one head.'
```
Comment (jtainslie) [inline src/modeling/models/mmt_classification_model.py]: Nit: this should ideally be a ValueError (go/pystyle#exceptions-decision), and `classification_heads` can just be a positional argument if a `None` default is invalid.

File: src/tasks/classification.py [hunk 3]
```diff
@@ -0,0 +1,335 @@
...
+from absl import logging
+import numpy as np
+import orbit
+import tensorflow as tf
+
+from official.core import base_task
+from official.core import config_definitions as cfg
+from official.core import task_factory
+from official.nlp.data import data_loader_factory
+from official.nlp.modeling import layers
```
Comment (jtainslie) [inline src/tasks/classification.py]: Nit: this can be all the same block (after sorting)

File: src/tasks/classification.py [hunk 4]
```diff
@@ -0,0 +1,335 @@
...
+
+
+METRIC_TYPES = frozenset(['accuracy', 'auc'])
+
+
+@dataclasses.dataclass
+class ClassificationConfig(cfg.TaskConfig):
+  """The classification model config."""
+
+  model: mmt.ClassificationModelConfig = mmt.ClassificationModelConfig()
```
Comment (jtainslie) [inline src/tasks/classification.py:44]: Just checking if `ClassificationModelConfig` is immutable.  Mutable values are not ideal for attribute defaults.  Same for `DataConfig` below.

**Review by jtainslie — APPROVED** [review]
Readability lgtm.

**Review by wqfcr-google — APPROVED** [review]
